syntax = "proto3";

package hello;

service HelloService {
    rpc SayHello (HelloRequest) returns (HelloResponse);
    rpc DetectFraud (FraudRequest) returns (FraudResponse);
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string greeting = 1;
}

message FraudRequest {
    string name = 1;
}

message FraudResponse {
    bool decision = 1;
}
